applications/asset_tracker/src/gps_controller/gps_controller.c

```diff
@@ -194,11 +195,11 @@ void gps_control_on_trigger(void)
 }
 
 /** @brief Configures and starts the GPS device. */
-int gps_control_init(gps_trigger_handler_t handler)
+int gps_control_init(struct k_work_q *work_q, gps_trigger_handler_t handler)
 {
 	int err;
 	struct device *gps_dev;
-
+	gps_work_q = work_q;
```

Comment:
Could do a null pointer check here and return `-EINVAL` if null

Reply:
done